.gitignore

```diff
@@ -119,4 +119,7 @@ env/
 ### Configuration files ###
 .env
 
-.vscode/
\ No newline at end of file
+.vscode/
+
+### Vagrant ###
+.vagrant/
```

Comment:
You should probably gitignore logs.txt so that people don't accidentally commit it after running the app in vagrant.